site/documentation/content/api/command-and-control-kafka/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Command & Control API for Kafka Specification"
+linkTitle: "Command & Control API for Kafka"
+weight: 419
+---
+
+The *Command & Control* API of Eclipse Hono&trade; is used by *Business Applications* to send commands to connected devices.
+
+Commands can be used to trigger actions on devices. Examples include updating a configuration property, installing a software component or switching the state of an actuator.
+<!--more-->
+
+Hono distinguishes two types of commands. The first type is *one-way* only. In this case the sender of the command does not expect the device to send back a message in response to the command. This type of command is referred to as a *one-way command* in the remainder of this page. One-way commands may be used to e.g. *notify* a device about a change of state.
+
+The second type of commands expects a *response* to be sent back from the device as a result of processing the command. In this case the response contains a *status* code which indicates whether the command could be processed successfully. If so, the response may also include data representing the result of processing the command. This type of command is plainly referred to as a *command* because it represents the default case.
+
+The Command & Control API for Kafka is an alternative to the [Command & Control API for AMQP]({{< relref "/api/command-and-control" >}}).
+With this API, clients send command messages to an Apache Kafka&reg; cluster instead of an AMQP Messaging Network.
```

Comment:
General explanations about Hono's Kafka-based APIs are now on a separate page. FMPOV it would make sense to add a reference to it like the one in the downstream APIs:

```
See [Kafka-based APIs]({{< relref "/api/kafka-api" >}}) for fundamental information about Hono's Kafka-based APIs.
The statements there apply to this specification.
```

Reply:
I've added that.